tests/data/in-carta/colas-lab/shrink_colas_lab_data_for_tests.py

```diff
@@ -0,0 +1,60 @@
+"""
+Shrink datasets from Colas Lab from IN Carta provided as collection of CSV's.
+
+Note: built to be run from CytoTable poetry dev environment from project base, e.g.:
+`poetry run python tests/data/in-carta/colas-lab/shrink_colas-lab_data_for_tests.py`
+"""
+
+import pathlib
+
+import duckdb
+from pyarrow import csv
+
+# set a path for local and target data dir
+SOURCE_DATA_DIR = "tests/data/in-carta/colas-lab/data"
+TARGET_DATA_DIR = "tests/data/in-carta/colas-lab"
+
+# build a collection of schema
+schema_collection = []
+for data_file in pathlib.Path(SOURCE_DATA_DIR).rglob("*.csv"):
```

Comment:
I thought colas lab data were excel (.xlsx) files?

We want cytotable to be a direct plugin to IN Carta output

Reply:
@gwaybio - thanks for checking on this! The data I have access to today is only a series of CSV files (no XLSX). I'll sync up with you on making sure the correct data was used here.

Reply:
Confirmed that CSV was the correct data for this use case.